ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/services/tracking/QueueBackedTrackerSpec.scala

```diff
@@ -95,63 +95,77 @@ class QueueBackedTrackerSpec
     }
 
     "input is submitted, and the queue has been completed" should {
-      "return an ABORTED error" in {
```

Comment:
I don't think these tests were correctly working, previously. The queue was being completed/failed twice and it was returning the same result twice. cc @pbatko-da @tudor-da